src/signals/settings/categories/components/StandardTextsField/StandardTextsField.tsx

```diff
@@ -25,9 +25,10 @@ import { orderStandardTexts } from '../utils'
 type Props = {
   name: string
   onChange: (standardTexts: StandardText[]) => void
+  statusOption?: string
```

Comment:
is a type statusOption

Reply:
all right, but I only need the key so I've set the prop to StatusOption['key]